python/paddle/incubate/optimizer/functional/bfgs.py

```diff
@@ -25,18 +32,18 @@
 
 
 def minimize_bfgs(
-    objective_func,
-    initial_position,
-    max_iters=50,
-    tolerance_grad=1e-7,
-    tolerance_change=1e-9,
-    initial_inverse_hessian_estimate=None,
-    line_search_fn='strong_wolfe',
-    max_line_search_iters=50,
-    initial_step_length=1.0,
-    dtype='float32',
-    name=None,
-):
+    objective_func: Callable[[Tensor], float],
```

Comment:
```suggestion
    objective_func: Callable[[Tensor], Tensor],
```

应该是个标量形式的 `Tensor` ，具体类型还是 `Tensor`